Datatypes

conorflemingATX opened the pull request:
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] All tests passed: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript: 30 points

**Checkpoint 2**
- [ ] The application works as it should: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript and the DOM API: 30 points

**Checkpoint 3**
- [ ] Use of React: 25 points
- [ ] Accesses an API: 25 points
- [ ] Proper use of documentation (commenting on code): 25 points
- [ ] The application functions as it should: 25 points

yousifS commented on 01week/datatypes.js:
```diff
@@ -0,0 +1,74 @@
+  }
+}
+
+whateverThree(0, 0);
```
what happens when i runt whateverThree(true, false)? What should the function print out?

conorflemingATX commented:
Sorry, that was a bit a silly mistake. I'll have to check my work a lot better before I upload them in future. Fixed and updated!

Thanks for being it to my attention btw!

Conor


Get Outlook for Android<https://aka.ms/ghei36>

________________________________
From: yousifS <notifications@github.com>
Sent: Thursday, October 5, 2017 1:04:01 AM
To: conorflemingATX/javascript-workbook
Cc: conorflemingATX; Author
Subject: Re: [conorflemingATX/javascript-workbook] Datatypes (#1)


@yousifS commented on this pull request.

________________________________

In 01week/datatypes.js<https://github.com/conorflemingATX/javascript-workbook/pull/1#discussion_r142853324>:

> +    return;
+  }
+}
+
+whateverTwo(1, 0);
+
+//Runs if two things are both not true.
+let whateverThree = function(arg1, arg2) {
+  if (arg1 && arg2) {
+    return;
+  } else {
+    console.log('OK!!!');
+  }
+}
+
+whateverThree(0, 0);


what happens when i runt whateverThree(true, false)? What should the function print out?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/conorflemingATX/javascript-workbook/pull/1#pullrequestreview-67274900>, or mute the thread<https://github.com/notifications/unsubscribe-auth/Aco28nA5-4f1cce7uRwyrwGRNhriqFdUks5spHFRgaJpZM4PueI0>.

yousifS commented on 01week/datatypes.js:
```diff
@@ -0,0 +1,74 @@
+
+//Prints datatype
+let printDatat = function(arg) {
+  console.log(typeof(arg));
```
This does not really tell me if arg is null, or arg is NaN.
